### PR TITLE
@/climatemappedafrica Hero page map colour codes

### LINKED ISSUE #992: Color Coding for the Map in ClimateMap Home Page

### PR DESCRIPTION
**Pull request by koechkevin**
## Description

As per the feedback given during the demo, it is a requirement that all maps on Climatemapped Africa be colour-coded to match the explore page. In the words of Eric _Climatemapped mirrors Pesa Yetu, the designs don't need to be exactly the same but Climatemapped should be an improvement of PesaYetu features._
This PR 
- Adds colour codes to the Hero Map component. 
- Ensures map is visible on mobile devices

Fixes #992 

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

## Screenshots
image
Screenshot 2024-11-07 at 12 20 39
Screenshot 2024-11-07 at 12 24 43

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by koechkevin** [comment]
> 💯 @koechkevin . How does hovering look like?

@kilemensi I retained the hover colour as is on PesaYetu and location name appears next to Legend/Map Key

**Comment by kilemensi** [comment]
> @kilemensi I retained the hover colour as is on PesaYetu and location name appears next to Legend/Map Key

🤔  ... If a region already has that color, I believe it won't make any difference @koechkevin ... Try setting the color to `choropleth color` but opacity of between `.6` - `.8`.

**Review by m453h — APPROVED** [review]
Looks good to me ✅

**Comment by kilemensi** [comment]
Looks good @koechkevin 

1. There border on the map changes the first time you hover on it. I don't think this is the intended design (it doesn't on PesaYetu site) https://github.com/user-attachments/assets/95915108-d7c1-4da8-8262-839515d492f9
2. Why two labels? Tooltip _and_ text
Screenshot 2024-11-08 at 09 42 15
3. Can we remove the **right** `margin`/`padding` on the map so that everything aligns nicely to the right from top of the page to bottom?
Screenshot 2024-11-08 at 09 46 21

**Review by kelvinkipruto — APPROVED** [review]
LGTM

**Comment by kilemensi** [comment]
If my comments above have been addressed @koechkevin, don't wait for my approval if others are happy with the PR.

**Pull request merged**
